charts/redpanda/templates/secrets.yaml

```
@@ -20,6 +20,7 @@ apiVersion: v1
 kind: Secret
```

If the user is going to manage this secret manually, they need a way to prevent the helm chart from failing when it tries to create the same secret.
```suggestion
{{- if .Values.auth.sasl.createSecret }}
kind: Secret
```